pom.xml

```diff
@@ -128,42 +128,44 @@
 
    <properties>
       <java.version>1.8</java.version>
-      <spring.boot.version>2.1.4.RELEASE</spring.boot.version>
-      <spring.cloud.version>Greenwich.RELEASE</spring.cloud.version>
+      <spring.boot.version>2.3.4.RELEASE</spring.boot.version>
+      <spring.cloud.version>Hoxton.SR7</spring.cloud.version>
+      <spring.plugin.core.version>2.0.0.RELEASE</spring.plugin.core.version>
 
       <snapshotDependencyAllowed>true</snapshotDependencyAllowed>
 
       <!-- Spring boot version overrides (should be reviewed with every boot
          upgrade) - START -->
       <!-- Newer versions needed than defined in Boot -->
       <!-- Fixed dependencies -->
-      <rabbitmq.http-client.version>2.1.0.RELEASE</rabbitmq.http-client.version>
-      <!-- Older versions needed than defined in Boot -->
-      <!-- Piggy back existing Eclipse CQs -->
-      <commons-lang3.version>3.7</commons-lang3.version>
-      <classmate.version>1.3.4</classmate.version>
-      <!-- Newer versions (tested 4.5.7) do not seem to work with RabbitMQ management plugin -->
-      <httpclient.version>4.5.6</httpclient.version>
+      <rabbitmq.http-client.version>3.5.0.RELEASE</rabbitmq.http-client.version>
+      <!-- Newer version contains new rule that we violate. Needs to be fixed first -->
+      <hibernate-validator.version>6.0.20.Final</hibernate-validator.version>
+      <!-- Newer community version does not support MySQL 5.6 anymore -->
+      <flyway-core.version>5.2.4</flyway-core.version>
       <!-- Spring boot version overrides - END -->
 
       <!-- Vaadin versions - START -->
       <vaadin.version>8.11.3</vaadin.version>
       <vaadin.spring.version>3.2.1</vaadin.spring.version>
       <vaadin.spring.addon.version>2.0.0.RELEASE</vaadin.spring.addon.version>
       <vaadin.plugin.version>${vaadin.version}</vaadin.plugin.version>
+      <vaadin.addon.vaadin-lazyquerycontainer.version>7.6.1.3</vaadin.addon.vaadin-lazyquerycontainer.version>
```

Comment:
lazyquerycontainer and flexibleoptiongroup are not used after Vaadin 8 migration

Reply:
Thanks for the hint, just removed it in the last commit.